drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -335,7 +513,7 @@ local function handle_set_hue(driver, device, cmd)
     device:send(req)
   else
     log.warn("Device does not support huesat features on its color control cluster")
- end
+  end
```

Comment:
nit: remove whitespace

Reply:
Was this in a previous commit? Looks like there isn't extra whitespace anymore

Reply:
still sees to be there, not sure. It's at the little extra space in front of `end`, not trailing whitespace

Reply:
nevermind, sorry about that. everything is as expected here :+1:

Reply:
Ok sweet!